apps/epic-web/src/pages/api/skill/[...skillRecordings].ts

```diff
@@ -1,14 +1,22 @@
 import SkillRecordings, {
+  IncomingRequest,
   SkillRecordingsOptions,
 } from '@skillrecordings/skill-api'
 import {nextAuthOptions} from '../auth/[...nextauth]'
+import {getToken} from 'next-auth/jwt'
+import {NextApiRequest} from 'next'
+import {getCurrentAbility, UserSchema} from '@skillrecordings/skill-lesson'
 
 export const skillOptions: SkillRecordingsOptions = {
   site: {
     title: process.env.NEXT_PUBLIC_SITE_TITLE,
     supportEmail: process.env.NEXT_PUBLIC_SUPPORT_EMAIL,
   },
   nextAuthOptions,
+  getAbility: async (req: IncomingRequest) => {
```

Comment:
this let's use use abilities in a contextual way inside of skill-api for stuff that needs authorization